src/sarif-viewer.test.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { AxeResults } from 'axe-core';
+import * as fs from 'fs';
+import { convertAxeToSarif } from './index';
+
+describe('Azure DevOps sarif viewer extension test', () => {
+    it('save sarif file to artifact to be shown by sarif viewer tab', () => {
+        const axeJSON: string = fs.readFileSync(
+            './src/test-resources/axe322-v2.dequemars-testsite.1554329251110.json',
+            'utf8',
+        );
+        const axeResultStub: AxeResults = JSON.parse(axeJSON) as AxeResults;
+        fs.mkdir('./test-results', () => {
```

Comment:
When you use a callback in a test like this without blocking the test on the callback's completion, it means that you're at the mercy of timing of the jest process as to whether the work in the callback will actually complete before the process terminates.

The easiest way to manage this would be to use `fs.mkdirSync` instead.

Reply:
Good catch